student-info-system/sis/test_data/create_40_sectionstudent.py

```diff
@@ -40,7 +40,7 @@ def createData():
             # sort of interesting...
```

Comment:
@bjmckenz is this being saved for later?

Reply:
yes. It's a measure of how many courses are denied to students by not meeting prereqs. In the real world it should decline. I was watching it, and thinking if I could select courses better to be more representative of real data.